packages/foreman/foreman/foreman.spec

```diff
@@ -391,6 +399,7 @@ Meta package with support for building RPMs in the Foreman release cycle.
 
 %files build
 %{_sysconfdir}/rpm/macros.%{name}-dist
+# TODO: ship SOURCE1
```

Comment:
do we have to? it's the only project that uses a `Gemfile`, others would need `gemspec` support.